client/src/parsec/functions.ts

```diff
@@ -3,8 +3,6 @@
 import {
   libparsec,
   InviteListItem,
```

Comment:
should not this be imported by /types too?

Reply:
There's no reason to import/export it in types.ts. `InviteListItem` will never be used by someone who uses the API, they will only use `InviteListItemUser` or `InviteListItemDevice`. Since it's only used internally, I'd rather not export it at all.